custom_components/myskoda/coordinator.py

```diff
@@ -79,8 +80,12 @@ def __init__(
         myskoda.subscribe(self._on_mqtt_event)
 
     async def _async_update_data(self) -> State:
-        vehicle = await self.myskoda.get_vehicle(self.vin)
-        user = await self.myskoda.get_user()
+        _LOGGER.debug("Performing scheduled update of all data for vin %s", self.vin)
+        try:
+            vehicle = await self.myskoda.get_vehicle(self.vin)
+            user = await self.myskoda.get_user()
+        except ClientError as err:
```

Comment:
out of scope here but I wonder if our myskoda API package should raise its own exceptions instead of letting `aiohttp` (and other) exceptions bubble up.

Reply:
Would be nice, but i checked HA's coordinator code and in the end they catch aiohttp's exceptions as well as stopgap